pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -445,51 +540,123 @@ func (p *PoolManager) vmWaitingCleanupLook() {
 	for _ = range c {
 		p.poolMutex.Lock()
 
-		configMap, err := p.kubeClient.CoreV1().ConfigMaps(p.managerNamespace).Get(context.TODO(), names.WAITING_VMS_CONFIGMAP, metav1.GetOptions{})
-		if err != nil {
-			logger.Error(err, "failed to get config map", "configMapName", names.WAITING_VMS_CONFIGMAP)
-			p.poolMutex.Unlock()
+		p.handleStaleLegacyConfigMapEntries(logger)
+
+		p.handleStaleMacEntries(logger)
```

Comment:
Don't call this "handle" it means nothing, say something about what it's done.

Reply:
changed to `heal` .. I know it's not much better ><